packages/synapse-react-client/src/components/DirectDownload.tsx

```diff
@@ -39,6 +40,7 @@ const DirectDownload: React.FunctionComponent<
     fileHandleId,
     displayFileName,
     onClickCallback,
+    stopPropagation = false,
```

Comment:
Optional new property added to prevent click events from propagating if needed. When the download button for an individual file is pressed, we don't wish to also add it to the bulk download selection.